backend/compat.py

```diff
@@ -1 +1,13 @@
+# Werk van Sil
 
+# for-loop
+# [] 
+
+
+def validate_appointment(data: dict):
+    KeyList = ['title', 'description', ...]
+    for key in KeyList:
```

Comment:
- X stelt één key uit de dictionary voor en y is die list. Dus zolang er nog een key in de dictionary zit die de loop nog niet gehad heeft gaat de loop door.
- Range genereerd nummers en daardoor kun je hem gebruiken voor loops.
- Enumerate geeft elke variabele uit een list een waarde beginnend bij 0.
- hij print de keys namelijk 1 en 3, de 2 en 4 zijn de values.
- 'for value in {1: 2, 3: 4}: print(value)'.
-  Hij print de waardes 1 2 3 en 4, deze items zitten in een list en hiervoor zaten de keys met values in een dictionary.(zit dus verschil tussen)